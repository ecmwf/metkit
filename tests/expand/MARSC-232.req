retrieve,
    area=85.000/-180.000/-60.000/180.000,class=od,date=20231003/to/20231003,expver=1,grid=0.100/0.100,levtype=sfc,padding=0,param=144.128,step=0/to/12/by/1,stream=oper,time=00/12,type=fc,dataset=mars,
    target=reference.data 