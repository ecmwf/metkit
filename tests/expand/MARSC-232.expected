RETRIEVE,
    DATASET    = mars,
    CLASS      = OD,
    TYPE       = FC,
    STREAM     = OPER,
    EXPVER     = 0001,
    REPRES     = GG,
    LEVTYPE    = SFC,
    PARAM      = 144,
    DATE       = 20231003,
    TIME       = 0000/1200,
    STEP       = 0/1/2/3/4/5/6/7/8/9/10/11/12,
    DOMAIN     = G,
    TARGET     = reference.data,
    RESOL      = AUTO,
    AREA       = 85/-180/-60/180,
    GRID       = .1/.1,
    PADDING    = 0 